.github/workflows/remove_template_files.yml

```diff
@@ -1,57 +1,38 @@
-# https://github.com/orgs/community/discussions/22183#discussioncomment-6321583
+# inspired by: https://github.com/orgs/community/discussions/22183#discussioncomment-6321583
 
-name: first-time-setup
-run-name: first time setup of repo
+name: Template Cleanup
+run-name: template-cleanup
 
 on:
-  # run when branch created (repo generated from template)
   create:
 
-# only keep latest run of this workflow
-concurrency:
-  group: first-time-setup
-  cancel-in-progress: true
-
-# ADDED THIS CHANGE
-permissions: 
-  actions: write
-  checks: write
-  contents: write
-
-
 jobs:
-  first-time-setup:
-    # ensure run only once, when repo generated
-    if: github.run_number == 1
-
+  template-cleanup:
+    name: template-cleanup
     runs-on: ubuntu-latest
+    
+    permissions:
+      contents: write
 
     steps:
-      # get main branch repo contents
-      - name: Checkout code
-        uses: actions/checkout@v3
-        # ADDED THIS CHANGE
-        with:
-          repo-token: ${{ secrets.GITHUB_TOKEN }}
-
-      # remove files not needed for user instance of template
-      - name: Remove unneeded files
+      - uses: actions/checkout@v3
+      
+      # Cleanup project
+      - name: Cleanup
         run: |
-          mv README.md notes.md
-          rm -rf .github
-          rm -f environment.yml
-          touch README.md
-          echo '# Generated from toolbox-workshop-protocol-template' >> README.md
-          echo 'https://github.com/pep-dortmund/toolbox-workshop-protocol-template' >> README.md
-
+            mv README.md notes.md
+            rm -rf .github
+            rm -f environment.yml
+            touch README.md
+            NAME="${GITHUB_REPOSITORY##*/}"
+            echo "# $NAME" >> README.md
+            echo 'Generated from [toolbox-workshop-protocol-template](https://github.com/pep-dortmund/toolbox-workshop-protocol-template)' >> README.md
```

Comment:
We should think about also adding a hyperlink pointing to `notes.md` in the README. I'm thinking of something like
```md
[Read the notes](notes.md) on how to use this template.
```